assembl/static/js/app/views/baseMessageList.js

```diff
@@ -731,12 +731,12 @@ return cls.extend({
         return true;
       });
     }).catch(function(e) {
-      if (raven_url) {
+      if (sentry_dsn) {
```

Comment:
Raven? it's removed?

Reply:
Raven is the old sentry sdk. After merge, we will still use it for v1 js but not for python and static2. Here it is the variable from the rc/ini files

Reply:
cool ok :+1: